lib/rules/comment-word-disallowed-list/__tests__/index.mjs

```diff
@@ -331,7 +331,7 @@ testRule({
 			code: '// TODO:',
 			message: messages.rejected('/^TODO:/'),
 			line: 1,
-			column: 4,
+			column: 1, // should be `4`, see: https://github.com/shellscape/postcss-less/issues/178
```

Comment:
`postcss-less` has a bug with the source position of inline comments.

It reports a range that is too short and possibly the described range doesn't include the `word` that is being searched for.

When `word` isn't found, the fallback used by PostCSS is the entire range of the node, which is still too short.

https://github.com/shellscape/postcss-less/issues/178